assets/js/base/components/cart-checkout/totals/shipping/shipping-via.tsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const ShippingVia = ( {
+	selectedShippingRates,
+}: {
+	selectedShippingRates: string[];
+} ) => {
+	return (
+		<div className="wc-block-components-totals-item__description wc-block-components-totals-shipping__via">
```

Comment:
Is a dedicated classname needed for these new descriptions?

Reply:
There is a way to target this element without adding the dedicated class, but the CSS rule looks like this.

`.wc-block-components-totals-shipping .wc-block-components-totals-item__description > .wc-block-components-totals-item__description`

I'm not such a fan of it and would prefer to keep the dedicated class name.